website/docs/r/service_aws.html.markdown

```diff
@@ -21,3 +21,9 @@ The following arguments are supported:
 * `repo` - (Required) The path of the repository on which the service operates.
 * `role` - (Required) The role name or ARN of the IAM role that should have access to this service account.
 
+## See also
+
+- [SecretHub Amazon Web Services Integration](https://secrethub.io/docs/reference/aws/)
```

Comment:
How about `AWS Integration` as a title? That's more in line with the other titles.

Reply:
Sure, it sounds better. I'll change it!